components/Card.js

```diff
@@ -18,6 +17,7 @@ export default function GridOfCards() {
     });
     setCards(updatedCards);
```

Comment:
I'd prefer you didn't use the setter function `setCards` and instead defined a handler function at the top level of your app, like `app.js`. The handler could look like this:

```js
function handleUpdateCards ({updatedCards}) {
setCards(updatedCards)
}
```

You'd then use the handler instead of the setter in your `handleActive` function. It's the same pattern you used for your `submitNewCard` function.

By the way, I'd probably suggest a name such as `toggleActiveCard` for the `handleActive` function. 🤔 It's just toggling a boolean value, as far as I can tell.

Reply:
Hey, CJ, I will uplift the function in US 3 - I hope that's fine.